ogre2/include/ignition/rendering/ogre2/Ogre2DepthCamera.hh

```diff
@@ -118,6 +122,28 @@ namespace ignition
       /// ogre camera has not been created.
       public: double FarClipPlane() const override;
 
+      // Documentation inherited.
+      // TODO(anyone): this function should be virtual, declared in 'Camera'
+      // and 'BaseCamera'. We didn't do it to preserve ABI.
+      // Looks in commit history for '#SetShadowsNodeDefDirtyABI' to
+      // see changes made and revert
+      public: void SetShadowsNodeDefDirty();
+
+      // TODO(anyone): This fixes the pass quad material leaving dangling
+      // pointers when we remove the workspace, so we have to cleanup the
+      // material first.
+      // This bug was fixed in Ogre 2.2 thus the workaround should not be
+      // necessary there
+      //
+      // See https://github.com/ignitionrobotics/ign-rendering/
+      // pull/303#pullrequestreview-635228897
+      // Repro:
+      //  1.run the sensors_demo.sdf world: ign gazebo -v 4 -r sensors_demo.sdf
+      //  2.open Lights plugin, top right menu (3 dots) and select Lights
+      //  3.insert a spot light into the scene (needs to be spot light)
+      //  4.ign gazebo crashes
+      private: void removeWorkspaceCrashWorkaround();
```

Comment:
can you change capitalize this function, i.e. `RemoveWorkspaceCrashWorkaround`? private functions are also capitalized in Igntiion.

Reply:
Aggh, sorry. Force of habit.

I'm aware of this style but I've unconsciously wrote it that way. I'll fix them